### ISSUE TITLE
[css-toggles] Clarify what "persistent state" means

### ISSUE THREAD

**Issue by miketaylr** [w3c/csswg-drafts#7231]
In https://tabatkins.github.io/css-toggle/#toggles (and elsewhere), the notion of "persistent state" is used:

> [Toggles](https://tabatkins.github.io/css-toggle/#css-toggle) are persistent state on an element, and are not directly affected by any CSS properties.

It may be useful to define this, or clarify in a security and privacy considerations section that toggles _aren't_ cookies, or storage. Or to define what persistent state means so it's more obvious.

**Comment by tabatkins** [comment]
Indeed, this is meant to be page-lifetime state, same as the contents of a form element. It can persist in the BF cache but no further; anything more requires script to actually persist it somehow. The lifetime considerations aren't defined at all right now, which needs fixing.
